### PR TITLE
Update mint only if it belongs to MCC

### PR DESCRIPTION
**Pull request by imabdulbasit**

### CODE REVIEW

File: api/src/mutations/mint.rs [hunk 1]
```diff
@@ -404,6 +404,7 @@ impl Mutation {
             created_by: Set(user_id),
             compressed: Set(compressed),
             credits_deduction_id: Set(Some(credits_deduction_id)),
+            edition: Set(-1),
```
Comment (kespinola) [inline api/src/mutations/mint.rs:407]: Thought the default was already -1.

Comment (kespinola) [reply api/src/mutations/mint.rs:407]: confirmed.

**Comment by kespinola** [comment]
https://twitter.com/0xPrismatic/status/1692198401918615888

File: api/src/mutations/mint.rs (hunk 1 above)
Comment (imabdulbasit) [reply api/src/mutations/mint.rs:407]: yeah sorry I had to remove this and merge it but I forgot

**Pull request closed without merging**